dev_guide/builds/triggering_builds.adoc

```diff
@@ -32,6 +32,12 @@ link:https://docs.gitlab.com/ce/user/project/integrations/webhooks.html[GitLab],
 link:https://confluence.atlassian.com/bitbucket/manage-webhooks-735643732.html[Bitbucket],
 or Generic webhooks.
 
+[NOTE]
+====
+`oc new-app` and `oc new-build` will create GitHub and Generic webhook triggers
+for you, but you must add any other needed webhook triggers manually (see xref:describe-buildconfig[Displaying Webhook URLs]).
```

Comment:
as a general rule in technical documentation, I prefer to avoid referring to the reader directly as it comes across as somewhat informal/casual. suggest:

"....will create GitHub and Generic webhook triggers automatically, but any other webhook triggers must be added manually. (see ....." 

instead.